package.json

```diff
@@ -4,13 +4,14 @@
   "private": true,
   "scripts": {
     "android": "react-native run-android",
-    "ios": "react-native run-ios",
+    "ios": "react-native run-ios  --simulator='iPhone 8'",
```

Comment:
simulator should be something you add by yourself, not here

Reply:
remove it

Reply:
fixed